circuit/pallets/execution-delivery/src/benchmarking.rs

```diff
@@ -161,7 +161,7 @@ fn insert_default_xdns_record<T: pallet_xdns::Config>() {
 
     let gateway_genesis = GatewayGenesisConfig {
         modules_encoded: None,
-        signed_extension: None,
+        // signed_extensions: None,
```

Comment:
why commented out?

Reply:
This one was breaking the decoding of payloads from js clients and is also unused in the codebase, so I commented it out as a workaround until we need it for some other implementation.